bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/auxfile/AuxFileEditor.scala

```diff
@@ -27,7 +27,7 @@ object AuxFileEditor {
       lab.text = size.toString
   })
 
-  private val Format = new SimpleDateFormat("MM/dd/yy HH:mm:ss") {
+  private val Format = new SimpleDateFormat("yyyy-MMM-dd HH:mm:ss") {
```

Comment:
It looks like it should be straightforward.  You can add a bit to the "Start" and "End" columns [here](https://github.com/gemini-hlsw/ocs/blob/develop/bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/EdProgramHelper.scala#L25-L29) and take away a corresponding amount from "Keys Used".  I think it is worth doing.